src/main/java/edu/kis/powp/jobs2d/drivers/UsageMonitor.java

```diff
@@ -0,0 +1,75 @@
+package edu.kis.powp.jobs2d.drivers;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+import java.text.DecimalFormat;
+import java.util.logging.Logger;
+
+import static java.lang.Math.pow;
+import static java.lang.Math.sqrt;
+
+public class UsageMonitor implements Job2dDriver {
```

Comment:
This class uses a design pattern. It would be good to add its name to the class name.